lib/support/tcpdump.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { spawn } = require('child_process');
+const get = require('lodash.get');
+const mkdirp = require('mkdirp');
+
+/**
+ * Create a new TCPDUMP instance
+ * @class
+ */
+class TCPDUMP {
+  constructor(directory, options, iteration) {
+    this.directory = directory;
+    this.options = options;
+    this.iteration = iteration;
+  }
+
+  start() {
+    const useTcpDump = get(this.options, 'pcap', false);
+    if (useTcpDump === true || useTcpDump === 'true') {
+      // Create Capture dir
+      const capture_dir = this.directory + '/capture';
+      mkdirp(capture_dir);
+
+      // Start TcpDump
+      // Sould set suid bit with "sudo chmod +s /usr/sbin/tcpdump" if running outside docker
+      const capture_file = capture_dir + '/' + this.iteration + '.pcap';
```

Comment:
You could probably use path join here to join the dirs (https://nodejs.org/api/path.html#path_path_join_paths) then at least in theory it will work on Windows. We don't support Windows but we try to keep the code working.